doc/source/whatsnew/v0.23.0.txt

```diff
@@ -176,6 +176,37 @@ Please note that the string `index` is not supported with the round trip format,
    new_df
    print(new_df.index.name)
 
+.. _whatsnew_0230.enhancements.index_division_by_zero
+
+Index Division By Zero Fills Correctly
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Division operations on ``Index`` and subclasses will now fill positive / 0 with ``np.inf``, negative / 0 with ``-np.inf``, and 0 / 0 with ``np.nan``.  This matches existing Series behavior.
```

Comment:
double backticks on Series. Add the issue number, can be multiple (e.g. this PR number and the xref issue)